### PR TITLE
Quiesce exceptions

### PR DESCRIPTION
**Pull request by tomnis**
- this should cut down on spurious stacktraces in logs

### CODE REVIEW

**Comment by tomnis** [comment]
let me also change the influxdb version we are using

File: src/test/scala/com/workday/warp/arbiters/SmartNumberArbiterSpec.scala [hunk 1]
```diff
@@ -36,7 +36,8 @@ class SmartNumberArbiterSpec extends WarpJUnitSpec with CorePersistenceAware {
   def testSmartNumberBehavior(): Unit = {
     val arbiter: SmartNumberArbiter = new SmartNumberArbiter
     // create 1000 gaussian numbers with mean 50
-    val responseTimes: List[Double] = List.fill(1000)(50 + (Random.nextGaussian * 4))
+    val random: Random = new Random(seed = 123456)
```
Comment (mottati) [inline src/test/scala/com/workday/warp/arbiters/SmartNumberArbiterSpec.scala]: Why are we adding this?

File: src/main/scala/com/workday/warp/common/HasCoreWarpProperties.scala [hunk 2]
```diff
@@ -91,6 +91,16 @@ trait HasCoreWarpProperties extends WarpPropertyLike {
...
 
+  /**
+    * Whether MeasurementCollectors should log stacktraces when errors occur, or just an error message.
+    *
+    * Note that setting this to true can result in an overly verbose build log.
+    *
+    * Required: No
+    * Default Value: false
+    */
+  val WARP_LOG_STACKTRACES: PropertyEntry = PropertyEntry("wd.warp.log.stacktraces", isRequired = false, "false")
```
Comment (rooeque) [inline src/main/scala/com/workday/warp/common/HasCoreWarpProperties.scala]: should we go more granular in our log naming? i.e `wd.warp.log.mc.stacktraces`

Comment (tomnis) [reply src/main/scala/com/workday/warp/common/HasCoreWarpProperties.scala]: cool ill rename it

File: src/test/scala/com/workday/warp/arbiters/SmartNumberArbiterSpec.scala (hunk 1 above)
Comment (tomnis) [reply src/test/scala/com/workday/warp/arbiters/SmartNumberArbiterSpec.scala]: i thought it would be nice to have a deterministic random number generator

**Pull request merged**